modules/data.atmosphere/inst/scripts/runtest-download.NOAA_GEFS.R

```diff
@@ -0,0 +1,38 @@
+#This program is to test the R script donload.NOAA.R during development.
```

Comment:
Again, this shouldn't hold up the PR, but it would be nice to convert this to some kind of [unit test](http://r-pkgs.had.co.nz/tests.html).